### DISCUSSION TITLE
Client-side imports possible?

### DISCUSSION THREAD

**Discussion by vicary** [webpack/webpack#16615] [Q&A] [answered]
We have some 3rd party pre-built ESM modules made for Web Workers, we would like to `await import(...);` in the browser environment.

Is it possible to "escape" some of the `import(...)` statements and have webpack bundle it as-is, such that it runs at client side?

**Comment by r1m** [comment, marked as answer]
`import(/* webpackIgnore: true */"...")`

**Reply by vicary** [reply]
Thanks!
